config/dev/aws-managedcluster.yaml

```diff
@@ -13,7 +13,7 @@ spec:
       instanceType: t3.small
     controlPlaneNumber: 1
     publicIP: true
-    region: us-west-2
+    region: ${AWS_REGION}
```

Comment:
Let's use [well-known variable names](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html) instead of inventing our own.

Default value for this one should be set to `us-east-2`

Reply:
Making this unconfigurable is a bad practice, why not expose it via environ with some sane default value so that anyone could set custom region without manually editing _some_ YAMLs?

P.S. Any other way of changing hard-coded region is also OK.

Reply:
I'm ok with making it configurable. I'm not ok with: bad naming; absence of default value (in Makefile).

Reply:
Didn't invent a new name. Got this from clusterawsadm output: 

```
$ clusterawsadm bootstrap iam create-cloudformation-stack
AWS_REGION env not set and --region flag not provided, default configuration will be used
```

More info [here](https://docs.aws.amazon.com/sdkref/latest/guide/feature-region.html) and [here](https://stackoverflow.com/questions/59961939/what-is-the-difference-between-aws-default-region-and-aws-region-system-variable).

Set the default value to us-east-2.
(although I am against this personally; its good to ask the user to set the region that they want instead of defaulting to something.)

Reply:
I'm convinced. Thanks for going extra mile and explaining it